### ISSUE TITLE
Deprecation warning when populateFormValues a Category field

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#592]
**Description**
I got a deprecation warning when `populateFormValues` a Category field:
```
{% do craft.formie.populateFormValues(form, {
	myFormieCategoriesField: currentUser.myUserCategoriesFieldHandle.ids()
}) %}
```

`Looping through element queries directly has been deprecated. Use the all() function to fetch the query results before looping over them.`

When I remove the `populateFormValues` setting, and if I simply print the `currentUser.myUserCategoriesFieldHandle.ids()` value, the deprecation warning is not showing again. So the warning really seems to come from Formie.

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.4.19
- Craft version: 3.7.14
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.20](https://github.com/verbb/formie/releases/tag/1.4.20)
